angular/wolox-books/src/app/screens/register/register.component.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../scss/colors';
+
+.register-form {
+  align-items: center;
+  background-color: $wild-sand;
+  border-top: 4px solid $blue;
+  display: flex;
+  flex-direction: column;
+  left: 50%;
```

Comment:
instead of doing the fixed position and left 50 top 50 you can just tell the parent element to have display flex, juestify content center and align items center